lib/marginalia.rb

```diff
@@ -48,14 +48,23 @@ def self.included(instrumented_class)
     def annotate_sql(sql)
       Marginalia::Comment.update_adapter!(self)
       comment = Marginalia::Comment.construct_comment
-      if comment.present? && !sql.include?(comment)
-        sql = "#{sql} /*#{comment}*/"
-      end
       inline_comment = Marginalia::Comment.construct_inline_comment
-      if inline_comment.present? && !sql.include?(inline_comment)
-        sql = "#{sql} /*#{inline_comment}*/"
```

Comment:
Before you could have a normal comment + add an inline comment.
I think your new implementation will only have one or the other.
cc @mattyoho to double check if what I am saying make sense.

Reply:
Yep, I think what you're saying makes sense, @arthurnn. Both comments need to be escaped and concatenated together when present. :+1:

Reply:
Seems to have been a brain fart. You are both right, and I've fixed it.